### PR TITLE
Added zsh completion

### PR DESCRIPTION
**Pull request by dopsi**
I added a zsh completion function.

Let me know if you want me to put it in another location.

### CODE REVIEW

**Comment by ilpianista** [comment]
Great!

I would like to put all these stuff (completions, systemd scripts,...) to a 'extras' folder, but I can do that myself later.

File: completions/zsh/_arch-audit [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#compdef arch-audit
+
+typeset -A opt_args
+
+local context state line
+
+_arguments -s -S \
+"(-h --help)"{-h,--help}"[Prints help information]" \
+"(-q|--quiet)"{-q,--quiet}"[Show only vulnerable package names and their versions.]" \
```
Comment (ilpianista) [inline completions/zsh/_arch-audit]: I guess `-q|--quite` is a typo?

Also, what about `-qq` it's possible to have?

**Comment by ilpianista** [comment]
Thanks! If you merge both commits I'll push it.

**Comment by dopsi** [comment]
Before I merge all commits together, I would like to know if I have to restrict the argument of the `--dbpath` to a directory (it seems it is the expected argument) or if it can also be a file.

**Comment by ilpianista** [comment]
That should be a directory

**Comment by ilpianista** [comment]
As soon as you squash all your commits I merge your pull request.

**Comment by dopsi** [comment]
It's done. I actually had to learn how to squash commits together.

Let me know if you want me to update the completion function once you add new options and flags.

**Pull request merged**